src/actions/account.js

```diff
@@ -405,16 +410,17 @@ export const fundCreateAccountLedger = (accountId, ledgerPublicKey) => async (di
 }
 
 // TODO: Refactor common code with setupRecoveryMessageNewAccount
-export const handleCreateAccountWithSeedPhrase = (accountId, recoveryKeyPair, fundingOptions) => async (dispatch) => {
-    if (DISABLE_CREATE_ACCOUNT && !fundingOptions) {
+export const handleCreateAccountWithSeedPhrase = (accountId, recoveryKeyPair, fundingOptions, recaptchaToken) => async (dispatch) => {
```

Comment:
nit: shouldn't block on it right now, but seems like `recaptchaToken` can be part of `fundingOptions` (and then you won't have to pass it as separate argument downstream)

Reply:
Thanks for the feedback!  Just to talk through my reasoning on this, I had considered doing this, but `fundingOptions` conceptually holds data that the client uses to create funded accounts using direct calls to the NEAR network itself, from the browser, whereas `recaptchaToken` is only relevant when we are delegating the account creation to the backend contract-helper `/fundedAccount` endpoint, so I felt like it would be confusing to lump them together.

Also, until now `fundingOptions` in these functions could be traced directly to the URL arguments, but adding `recaptchaToken` to it would mean that the lifecycle of the data contained therein would be more complicated to trace, because we really don't want someone to be able to pass `recaptchaToken` in as a URL argument under any circumstances so would have to add it in somewhere upstream.  I also thought that the fact that the content of the `fundingOptions` object would need to contain either a single value 
 -- _only_ `reCaptchaToken` OR a _set_ of the other relevant `fundingOptions.xxxx` properties that are relevant to `near-api-js` calls made from the browser indicated that the recaptchaToken isn't really conceptually the same as the data currently in `fundingOptions`